### PR TITLE
Places part 1

### PR DESCRIPTION
**Pull request by orangewolf**
# TLDR
Rubocop and Rspec were added. This means a lot of files got "cleaned up" by rubocop. So we want to get this merged in pretty quickly to prevent conflicts. We also laid a lot of groundwork for the places features so we want to make sure we are not going to run in to complex merges down the line. This code could go to prod w/o breaking any interfaces. Nothing would show on the non-admin interface until you attached a place. Having said that, it should be tested in staging before going "live".

# More Details
- Dockerization - ability to spin app up in docker for development
- Rubocop set up and adoption
- Rspec setup and initial test for new features
- Reorder gemfile (Rubocop and Rob both feel strongly about this)
- Added Ancestry (for categories), Geocoder, pgSearch (for full text search of places)
- Initial admin crud for
  - comments
  - places
  - locations
  - categories
- Interface for adding places to stories

Interface to add categories to places, prep for adding categories to stories

Added map and geocoding for places

Cleaned up unused test suite

### CODE REVIEW

File: db/migrate/20210630032857_create_categories.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateCategories < ActiveRecord::Migration[6.1]
```
Comment (tkfarr) [inline db/migrate/20210630032857_create_categories.rb:1]: @orangewolf I see that this is commented out. Is this waiting on the renaming of `PlaceCategories` to `StoryPlaceCategories` to go live?

Comment (tkfarr) [reply db/migrate/20210630032857_create_categories.rb:1]: Nevermind, it was greyed out for some reason in my browser.

**Comment by sara-g** [comment]
Hi, do you have Tuple?  Want to pair to get these conflicts resolved/go over the changes?

**Comment by tkfarr** [comment]
@sara-g Hi Sara. Feel free to attack the merge conflicts. The only changes I made were in regards to naming PlaceCategories to PlaceGroupings and StoryPlaceCategory to PlaceGroupingsStory. This involved changing the MVC involving them. I'll review the latest commit later before merging.

**Comment by sara-g** [comment]
These 2 files:
app/controllers/admin/reports_controller.rb
app/controllers/admin/video_stories_controller.rb 
were not loading, so I fixed them to get our staging environment to load.  Those will be in the new MR we send.   I'll send you that later today.

If you made changes to the Schema AFTER our MR, I'm not sure what to do about those changes, but go with yours for now.  

The Place Category stuff we did not change.

**Comment by tkfarr** [comment]
Okay. Thanks for making the changes. Let me know if anything else comes up.
